include/envoy/registry/registry.h

```diff
@@ -130,9 +131,19 @@ template <class T, class Base> class RegisterFactory {
  * Macro used for static registration.
  */
 #define REGISTER_FACTORY(FACTORY, BASE)                                                            \
+  void forceStaticLink##FACTORY() {}                                                               \
```

Comment:
I think you didn't rename this one?

Reply:
ugh, fixed and recompiled just to be safe